features/private-network-access.yml

```
@@ -0,0 +1,7 @@
+name: Private Network Access
+description: The Private Network Access feature is a set of modifications to Fetch and HTML which are intended to mitigate the risks associated with unintentional exposure of devices and servers on a client’s internal network to the web at large.
```

```suggestion
description: Private Network Access modifies Fetch and HTML to mitigate the risks associated with unintentional exposure of devices and servers on a client’s internal network to the web at large.
```

---

The description talks about modifying other features. It's not clear how this is a feature to developers. I had a skim through the spec, so I don't really understand this thing, but I suspect we want to say something more in this direction:

```suggestion
description: The `Access-Control-Request-Private-Network: true` and `Access-Control-Allow-Private-Network: true` HTTP request and response headers establish permission for websites to make requests to internal networks, such as to routers.
```

Curiously, BCD doesn't appear to represent the request header. If we went this route, we probably ought to file a bug on BCD reporting it as missing.

---

Filed mdn/browser-compat-data/issues/25467

---

Committed this change, which maybe is enough until BCD is sorted.

---

Or we could leave this open and blocked until then.

---

As long as the `Request.targetAddressSpace` property is representative of browser support for this (which it appears to be, at least for now) we don't have to block on the missing keys. Let's go with this.